regression-test/suites/query_p0/sql_functions/aggregate_functions/test_aggregate_group_array.groovy

```diff
@@ -0,0 +1,261 @@
+            group_uniq_array(c_varchar),
+            group_uniq_array(c_string),
+            group_uniq_array(c_date),
+            group_uniq_array(c_datev2),
+            group_uniq_array(c_date_time),
+            group_uniq_array(c_date_timev2),
+            group_uniq_array(c_string_not_null)
+        FROM
+            ${tableName}
+    """
+
+    qt_select """
+        SELECT
+            group_uniq_array(c_bool,1),
+            group_uniq_array(c_tinyint,1),
+            group_uniq_array(c_smallint,1),
+            group_uniq_array(c_int,1),
+            group_uniq_array(c_bigint,1),
+            group_uniq_array(c_largeint,1),
+            group_uniq_array(c_float,1),
+            group_uniq_array(c_double,1),
+            group_uniq_array(c_decimal,1),
+            group_uniq_array(c_char,1),
+            group_uniq_array(c_varchar,1),
+            group_uniq_array(c_string,1),
+            group_uniq_array(c_date,1),
+            group_uniq_array(c_datev2,1),
+            group_uniq_array(c_date_time,1),
+            group_uniq_array(c_date_timev2,1),
+            group_uniq_array(c_string_not_null,1)
+        FROM
+            ${tableName}
+    """
+
+    qt_select """
+        SELECT
+            group_uniq_array(c_bool),
+            group_uniq_array(c_tinyint),
+            group_uniq_array(c_smallint),
+            group_uniq_array(c_int),
+            group_uniq_array(c_bigint),
+            group_uniq_array(c_largeint),
+            group_uniq_array(c_float),
+            group_uniq_array(c_double),
+            group_uniq_array(c_decimal),
+            group_uniq_array(c_char),
+            group_uniq_array(c_varchar),
+            group_uniq_array(c_string),
+            group_uniq_array(c_date),
+            group_uniq_array(c_datev2),
+            group_uniq_array(c_date_time),
+            group_uniq_array(c_date_timev2),
+            group_uniq_array(c_string_not_null)
+        FROM
+            ${tableName}
+        GROUP BY
+            c_id
+        ORDER BY
+            c_id
+    """
+
+    qt_select """
+        SELECT
+            group_uniq_array(c_bool,1),
+            group_uniq_array(c_tinyint,1),
+            group_uniq_array(c_smallint,1),
+            group_uniq_array(c_int,1),
+            group_uniq_array(c_bigint,1),
+            group_uniq_array(c_largeint,1),
+            group_uniq_array(c_float,1),
+            group_uniq_array(c_double,1),
+            group_uniq_array(c_decimal,1),
+            group_uniq_array(c_char,1),
+            group_uniq_array(c_varchar,1),
+            group_uniq_array(c_string,1),
+            group_uniq_array(c_date,1),
+            group_uniq_array(c_datev2,1),
+            group_uniq_array(c_date_time,1),
+            group_uniq_array(c_date_timev2,1),
+            group_uniq_array(c_string_not_null,1)
+        FROM
+            ${tableName}
+        GROUP BY
+            c_id
+        ORDER BY
+            c_id
+    """
+
+    sql """
```

Comment:
why we need this test and the below one?

Reply:
Just for group-by and non-group-by cases, which was in reference to the `test-aggregate-histogram`, if it makes no sense here, I'll remove them later.